src/main/java/com/github/scuwr/snitchvisualizer/handlers/ReflectionHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.github.scuwr.snitchvisualizer.handlers;
+
+import java.lang.reflect.Field;
+
+import net.minecraft.client.Minecraft;
+
+/**
+ * Because rendering shit in mc now requires reflection!
+ * 
+ * @author TealNerd
+ *
+ */
+public class ReflectionHandler {
```

Comment:
Yeah it's really stupid because you need those values to render anything in
relation to the player. But hey, the modding API is coming soon right
guys?!?!?!

On Saturday, May 16, 2015, Daniel Boston <notifications@github.com> wrote:

> In
> src/main/java/com/github/scuwr/snitchvisualizer/handlers/ReflectionHandler.java
> <https://github.com/Scuwr/Snitch-Visualizer/pull/4#discussion_r30465389>:
>
> > @@ -0,0 +1,26 @@
> > +package com.github.scuwr.snitchvisualizer.handlers;
> > +
> > +import java.lang.reflect.Field;
> > +
> > +import net.minecraft.client.Minecraft;
> > +
> > +/**
> > + * Because rendering shit in mc now requires reflection!
> > + *
> > + * @author TealNerd
> > + *
> > + */
> > +public class ReflectionHandler {
>
> I totally believe you but this is kind of a crazy move against modding on
> Minecraft's part. Ugh.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Scuwr/Snitch-Visualizer/pull/4/files#r30465389>.
>